FlowersStore/Utils/Layout/UIView+Constraints.swift

```diff
@@ -0,0 +1,212 @@
+        let constraint = bottomAnchor.constraint(equalTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func bottomAnchor(greaterThanOrEqualTo anchor: NSLayoutYAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = bottomAnchor.constraint(greaterThanOrEqualTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func bottomAnchor(lessThanOrEqualTo anchor: NSLayoutYAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = bottomAnchor.constraint(lessThanOrEqualTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func bottomAnchorToSuperview(constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required, safeArea: Bool = true) -> Self {
+        guard let superview = superview else { fatalError("View doesn't have a superview") }
+        let bottom: NSLayoutYAxisAnchor = safeArea ? superview.safeAreaLayoutGuide.bottomAnchor : superview.bottomAnchor
+        let constraint: NSLayoutConstraint = bottomAnchor.constraint(equalTo: bottom, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func leadingAnchor(equalTo anchor: NSLayoutXAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = leadingAnchor.constraint(equalTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func leadingAnchorToSuperview(constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        guard let superview = superview else { fatalError("View doesn't have a superview") }
+        let constraint = leadingAnchor.constraint(equalTo: superview.leadingAnchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func trailingAnchor(equalTo anchor: NSLayoutXAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
+        let constraint = trailingAnchor.constraint(equalTo: anchor, constant: constant)
+        constraint.priority = priority
+        constraint.isActive = true
+        return self
+    }
+
+    @discardableResult func trailingAnchor(greaterThanOrEqualTo anchor: NSLayoutXAxisAnchor, constant: CGFloat = 0, priority: UILayoutPriority = UILayoutPriority.required) -> Self {
```

Comment:
<!--
  0 failure: 
  1 warning:  Function 'trailin...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Utils/Layout/UIView+Constraints.swift;
  Line: 116;
-->

- :warning: Function 'trailingAnchor(greaterThanOrEqualTo:constant:priority:)' is unused